src/Umbraco.Cms.Integrations.Automation.Zapier/Controllers/PollingController.cs

```diff
@@ -98,23 +59,13 @@ public IEnumerable<PublishedContentDto> GetSampleContent()
 
         public List<Dictionary<string, string>> GetContentByType(string alias)
         {
-            var list = new List<Dictionary<string, string>>();
-
-            using (var cref = _umbracoContextFactory.EnsureUmbracoContext())
-            {
-                var cache = cref.UmbracoContext.Content;
-                
-                var node = cache.GetByXPath($"//{alias}")
-                    .Where(p => p.IsPublished())
-                    .OrderByDescending(p => p.UpdateDate)
-                    .FirstOrDefault();
+            if (!IsUserValid()) return null;
 
-                if (node == null) return list;
+            var contentType = _contentTypeService.Get(alias);
```

Comment:
Should have a null check here I think, in case the content type isn't found.  It would be unexpected, but still should consider here - and I guess return an empty dictionary.